internal/storage/storage.go

```diff
@@ -41,6 +41,9 @@ var ErrFileNotFound = errors.New("file not found")
 // ErrAccessDenied is returned when the service denies access. Either due to insufficient rights or wrong credentials.
 var ErrAccessDenied = errors.New("access denied")
 
+// ErrTooManyRequest is returned when the request number is exceeding rate limit.
+var ErrTooManyRequest = errors.New("too many requests. Your request exceeds the API rate limit. Please try it later")
```

Comment:
Keeping format in line with error message expectations:

```suggestion
var ErrTooManyRequest = errors.New("too many requests, please try again later")
```